requirements/doc.txt

```diff
@@ -54,11 +55,12 @@ jinja2==2.11.1
 lazy-object-proxy==1.4.3
 markupsafe==1.1.1
 mccabe==0.6.1
-mock==3.0.5
+mock==4.0.1
```

Comment:
This should be constrained to <4.0.0 to preserve Python 3.5 support.

Reply:
Added constraint as per suggestion.